client/src/main/java/io/pravega/client/stream/impl/StreamCutImpl.java

```diff
@@ -184,6 +186,37 @@ private void write00(StreamCutInternal cut, RevisionDataOutput revisionDataOutpu
         }
     }
 
+    // StreamCut serializer for version 1.
```

Comment:
Unrelated to the discussion below, by convention this should be called StreamCutSerializer10 not StreamCutSerializer01

Reply:
fixed it.